examples/bell-variational/main.py

```diff
@@ -18,9 +18,12 @@ def main(nshots, backend):
     set_backend(backend)
     initial_parameters = np.random.uniform(0, 2 * np.pi, 2)
     circuits = set_parametrized_circuits()
-    best, params, _ = optimize(
-        cost_function, initial_parameters, args=(circuits, nshots)
+
+    opt = ScipyMinimizer(
```

Comment:
This is still the old interface: we should move the loss function to `opt.fit(...)`.